### PR TITLE
Do not ever just copy things blindly into the json responses

### PR DESCRIPTION
**Pull request by aredridel**
This avoids copying internal data into json responses like it does for the template context. The template context is private but json responses are not.

### CODE REVIEW

**Comment by stubbornella** [comment]
@jefflembeck @jonathanmarvens - code review please!

File: adapters/bonbon.js [hunk 1]
```diff
@@ -94,20 +98,12 @@ exports.register = function(server, options, next) {
...
-        // appends charset to content-type header, for security reasons
-        request.response.type('text/html').charset('utf-8');
+    if (request.response.variety == 'view') {
+      request.response.source.context = Hoek.applyToDefaults(options, request.response.source.context);
+      request.response.source.context.user = request.loggedInUser;
 
-        break;
-      case "plain":
-        if (typeof (request.response.source) === "object") {
-          request.response.source = Hoek.applyToDefaults(options, request.response.source);
```
Comment (jonathanmarvens) [inline adapters/bonbon.js]: Is this really not needed, @aredridel?

Comment (aredridel) [inline adapters/bonbon.js]: Yeah, it's not actually used. Shocked me to find, but just not needed.

**Comment by jonathanmarvens** [comment]
LGTM, @aredridel.

**Pull request closed without merging**